src/core_plugins/input_control_vis/public/components/editor/list_control_editor.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { IndexPatternSelect } from './index_pattern_select';
+import { FieldSelect } from './field_select';
+
+export class ListControlEditor extends Component {
```

Comment:
Make stateless function.  I thought we had eslint checks for this recently put it place, but maybe it's only turned on in the uiFramework...  CJ would know but I don't want to tag him on paternity leave.

Few other files/components I see that should be in the functional stateless variety.